UnityProject/Assets/Scripts/UI/Systems/AdminTools/PlayerManagePage.cs

```diff
@@ -30,23 +30,23 @@ public class PlayerManagePage : AdminPage
 
 		[SerializeField] private Toggle OnWatchlist = null;
 
-		[SerializeField] private TMP_Text JailText;
+		[SerializeField] private Text JailText;
```

Comment:
TMP is what we want to use in the future, Text is considered deprecated due to its low fidelity when scaled up

Reply:
Made this change as the button text on the admin UI is small so the difference between TMP and regular text aint that much but its just more performant.
Also the rest of the UI is just regular text so the TMP stuff looks weird in contrast